R/robustbayesianmetaanalysis.R

```diff
@@ -909,37 +913,37 @@ RobustBayesianMetaAnalysis <- function(jaspResults, dataset, options, state = NU
-        y   = if (options[["measures"]] == "general" && options[["input_ES"]] != "")                    dataset[, .v(options[["input_ES"]])],
-        se  = if (options[["input_SE"]] != "")                                                          dataset[, .v(options[["input_SE"]])],
-        lCI = if (sum(unlist(options[["input_CI"]]) != "") == 2)                                        dataset[, .v(options[["input_CI"]][[1]][1])],
-        uCI = if (sum(unlist(options[["input_CI"]]) != "") == 2)                                        dataset[, .v(options[["input_CI"]][[1]][2])],
-        n   = if (options[["measures"]] %in% c("cohensd", "correlation") && options[["input_N"]] != "") dataset[, .v(options[["input_N"]])],
-        n1  = if (options[["measures"]] == "cohensd" && options[["input_N1"]] != "")                    dataset[, .v(options[["input_N1"]])],
-        n2  = if (options[["measures"]] == "cohensd" && options[["input_N2"]] != "")                    dataset[, .v(options[["input_N2"]])],
-        study_names = if (options[["input_labels"]] != "")                                              dataset[, .v(options[["input_labels"]])],
+        t   = if (options[["measures"]] == "cohensD" && options[["inputT"]] != "")                     dataset[, .v(options[["inputT"]])],
+        d   = if (options[["measures"]] == "cohensD" && options[["inputES"]] != "")                    dataset[, .v(options[["inputES"]])],
+        r   = if (options[["measures"]] == "correlation" && options[["inputES"]] != "")                dataset[, .v(options[["inputES"]])],
+        OR  = if (options[["measures"]] == "OR")                                                        dataset[, .v(options[["inputES"]])],
+        y   = if (options[["measures"]] == "general" && options[["inputES"]] != "")                    dataset[, .v(options[["inputES"]])],
+        se  = if (options[["inputSE"]] != "")                                                          dataset[, .v(options[["inputSE"]])],
+        lCI = if (sum(unlist(options[["inputCI"]]) != "") == 2)                                        dataset[, .v(options[["inputCI"]][[1]][1])],
+        uCI = if (sum(unlist(options[["inputCI"]]) != "") == 2)                                        dataset[, .v(options[["inputCI"]][[1]][2])],
+        n   = if (options[["measures"]] %in% c("cohensD", "correlation") && options[["inputN"]] != "") dataset[, .v(options[["inputN"]])],
+        n1  = if (options[["measures"]] == "cohensD" && options[["inputN1"]] != "")                    dataset[, .v(options[["inputN1"]])],
+        n2  = if (options[["measures"]] == "cohensD" && options[["inputN2"]] != "")                    dataset[, .v(options[["inputN2"]])],
+        study_names = if (options[["inputLabels"]] != "")                                              dataset[, .v(options[["inputLabels"]])],
```

Comment:
Don't you want to remove the underscores here?

Reply:
no, it's an argument to the `RoBMA` function exported from the `RoBMA` package